### ISSUE TITLE
Crash when using "Find Action..." Help menu item

### ISSUE THREAD

**Issue by BenLeggiero** [vsch/idea-multimarkdown#393]
```
java.lang.NullPointerException
	at com.vladsch.idea.multimarkdown.util.DelimitedBuilder.append(DelimitedBuilder.java:149)
	at com.vladsch.idea.multimarkdown.editor.actions.DisabledConditionBuilder.<init>(DisabledConditionBuilder.java:56)
	at com.vladsch.idea.multimarkdown.editor.actions.styling.ListItemAction.update(ListItemAction.kt:40)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareUpdate(ActionUtil.java:122)
	at com.intellij.ide.util.gotoByName.GotoActionModel.updateActionBeforeShow(GotoActionModel.java:267)
	at com.intellij.ide.util.gotoByName.GotoActionModel$ActionWrapper.getPresentation(GotoActionModel.java:510)
	at com.intellij.ide.util.gotoByName.GotoActionModel.lambda$updateActions$1(GotoActionModel.java:430)
	at com.intellij.util.containers.TransferToEDTQueue.processNext(TransferToEDTQueue.java:99)
	at com.intellij.util.containers.TransferToEDTQueue.access$300(TransferToEDTQueue.java:37)
	at com.intellij.util.containers.TransferToEDTQueue$1.run(TransferToEDTQueue.java:58)
	at com.intellij.openapi.application.TransactionGuardImpl$6.run(TransactionGuardImpl.java:318)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.runNextEvent(LaterInvocator.java:417)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:401)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:843)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:679)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:391)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by vsch** [comment]
@BenLeggiero, this issue appears to be a duplicate of #389 which was fixed in the latest release 2.3.3.

Could you please confirm your plugin version. You can see it in settings:

![Screen Shot Settings Version](https://github.com/vsch/idea-multimarkdown/raw/master/assets/images/faq/ScreenShot_Settings_Version.png)

**Comment by BenLeggiero** [comment]
![idea markdown settings](https://cloud.githubusercontent.com/assets/10189808/23415663/f2dcd80a-fdad-11e6-8529-e8693e5a2f2f.PNG)

**Comment by vsch** [comment]
@BenLeggiero, thank you. Yes it is the same issue that was fixed in version 2.3.3. You should upgrade to the latest to get the fixes and new functionality such as direct image pasting into documents to insert a link and save image file and PDF export.

**Comment by BenLeggiero** [comment]
Thanks, @vsch. Odd it didn't auto-update. Seems to be fixed now!
